xyz.parlatype.Parlatype.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "xyz.parlatype.Parlatype",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "parlatype",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--talk-name=org.freedesktop.secrets",
+        "--own-name=org.mpris.MediaPlayer2.parlatype"
+    ],
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/python3.8",
+        "/libexec",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "sphinxbase",
+            "config-opts": [
+                "--without-python"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/cmusphinx/sphinxbase/archive/9e901850ee4cd1c0b22f2d3b3c15f162564ca283.zip",
+                    "sha256": "4369fc4416b2bd3a396ca791cee1ca62c7360f549a78a3c754bb49ba0ebc843a"
+                }
+            ],
+        "cleanup": [
+            "/share/sphinxbase"
+            ]
+        },
+        {
+            "name": "pocketsphinx",
+            "config-opts": [
+                "--without-python"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/cmusphinx/pocketsphinx/archive/ab6d6471800966990e12fdb6ed27ae36323cf2c4.zip",
+                    "sha256": "29ce0c81eda60bef919d42287bcf85d29b8dcb8dcb8dc69998f58a79a6e4a5c0"
+                }
+            ],
+        "cleanup": [
+            "/share/pocketsphinx"
+            ]
```

Comment:
here too

Reply:
Fixed in https://github.com/flathub/flathub/pull/5312/commits/2be6165bb6284ca4562210d711c88b27cd6eff66